<launch>
    <node name="joint_trajectory_action_servery.py" pkg="baxter_interface" type="joint_trajectory_action_server.py" output="screen" launch-prefix="xterm -e" />
</launch>
